### PR TITLE
adapting values for GCP infrastructure, not all checks are covered by…

### PR DESCRIPTION
**Pull request by pirat013**
… the vars file and they need to be adapted in addition

### CODE REVIEW

File: runner/ansible/vars/gcp/10-default.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+
+expected_core:
+  "1.1.1": "20000"
+  "1.1.2": "" # unkown value, default may apply
```
Comment (arbulu89) [inline runner/ansible/vars/gcp/10-default.yml:5]: Typo `unkown` -> `unknown`.
Like in https://github.com/trento-project/trento/pull/825 I would remove the `default may apply` comment or add some real value (no empty)

File: runner/ansible/vars/gcp/10-default.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "1.1.3": "20"
+  "1.1.4": "60"
+  "1.1.5": "10"
+  "1.1.6": "udpu"
+  "1.1.7": "2"
+  "1.1.8": "1"
+  "1.2.1": "true"
+  "1.2.2.fence_gcpstonith": "300" # new variable
```
Comment (arbulu89) [inline runner/ansible/vars/gcp/10-default.yml:13]: Not harmful, but this variable won't do anything by now. I would be for removing it, and put it once we have the test updated (or a new one added most probably), so the new PR will make more sense

File: runner/ansible/vars/gcp/10-default.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  "1.3.2": "" # doesn't apply to gcp
+  "1.3.4": "" # doesn't apply to gcp
+  "1.3.5": "" # doesn't apply to gcp
+  "1.3.6": "" # doesn't apply to gcp
+  "2.2.1": "SLES_SAP"
+  "2.2.2": "15.1"
+  "2.2.3": "2.0.1"
+  "2.2.3.exclude": "2.0.3+20200511.2b248d828"
+  "2.2.4": "2.4.5" # unkown value, default may apply
+  "2.2.5": "1.4.0" # unkown value, default may apply
```
Comment (arbulu89) [inline runner/ansible/vars/gcp/10-default.yml:24]: Value `"2.2.5.exclude"` for missing

**Review by arbulu89 — CHANGES_REQUESTED** [review]

**Comment by pirat013** [comment]
@arbulu89  Some checks were not successful
Is there something I can fix?

File: runner/ansible/vars/gcp/10-default.yml (hunk 1 above)
Comment (pirat013) [reply runner/ansible/vars/gcp/10-default.yml]: Fixed

File: runner/ansible/vars/gcp/10-default.yml (hunk 2 above)
Comment (pirat013) [reply runner/ansible/vars/gcp/10-default.yml]: Done

File: runner/ansible/vars/gcp/10-default.yml (hunk 3 above)
Comment (pirat013) [reply runner/ansible/vars/gcp/10-default.yml]: Added

**Comment by stefanotorresi** [comment]
implemented in https://github.com/trento-project/runner/pull/34

**Pull request closed without merging**